### DISCUSSION TITLE
Amcrest 4k Camera 3D Printed Wall Bracket with drain

### DISCUSSION THREAD

**Discussion by SubBass100** [aaronwmorris/indi-allsky#949] [Show and tell]
I designed the wall mount bracket which you can download here: https://www.thingiverse.com/thing:6222878 
It is designed to be attached to a wall and wiring goes straight through to the inside. There are drainage holes to stop water from getting inside.

Amcrest 4k Camera with starlight sensor: https://www.amazon.com/gp/product/B083KMWCKJ/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

Glass Dome: https://www.amazon.com/gp/product/B09PHBP19F/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1 I use Rain-X on the inside and outside of it. The water beads right off.

Video: https://www.youtube.com/watch?v=wOo3-mLgGOk

![featured_preview_1c45c51f-61d5-4724-833c-99eafb3e55fb](https://github.com/aaronwmorris/indi-allsky/assets/60083876/58a53fc7-c0ff-4094-8daf-260c7e019e8a)

![featured_preview_e8ccd0a9-b205-4f4b-9e32-c84dfe357a74](https://github.com/aaronwmorris/indi-allsky/assets/60083876/89bba3f4-18a0-4345-850c-640741c25251)

![featured_preview_f69512b4-20a6-408f-ab31-3365f0d49094](https://github.com/aaronwmorris/indi-allsky/assets/60083876/11968419-365d-4171-b506-4cded812b807)

![featured_preview_df789a31-ce85-4d4f-858b-c063b86b2bca](https://github.com/aaronwmorris/indi-allsky/assets/60083876/c9b60516-c0fd-4463-9566-5d03cbfa9a81)

**Comment by aaronwmorris** [comment]
That is very interesting!  Do you have any pictures from the camera?

**Reply by SubBass100** [reply]
Here is a Time lapse Video from it.  Make sure you set the quality to the highest https://www.youtube.com/watch?v=wOo3-mLgGOk

![Screenshot_20230810_222720_DW Spectrum](https://github.com/aaronwmorris/indi-allsky/assets/60083876/b6421e01-6171-4b2c-b812-a74ea387b4d3)
![Screenshot_20230810_164848_DW Spectrum](https://github.com/aaronwmorris/indi-allsky/assets/60083876/a24c368c-3d32-48de-9271-81e24a79b8c5)

These images are screencaps from my phone but I think they show what it can see.

**Reply by aaronwmorris** [reply]
That looks very cool and unique.  I may investigate more with creating dark frames from JPEG sources like this.  It would remove the fixed pattern noise you see in the frame.

**Comment by aaronwmorris** [comment]
As of #1047 I have integrated being able to properly generate dark frames from JPEG sources such as IP web cameras.  I do not know how well the process will really work, but I think it might be an interesting possibility.
